src/Lender.sol

```diff
@@ -648,19 +650,27 @@ contract Lender {
     /// @param m maturity (timestamp) of the market
     /// @param a amount of underlying tokens to lend
     /// @param r slippage limit, minimum amount to PTs to buy
-    /// @param d deadline is a timestamp by which the swap must be executed
+    /// @param g guess parameters for the swap
+    /// @param market contract that corresponds to the market for the PT
     /// @return uint256 the amount of principal tokens lent out
     function lend(
         uint8 p,
         address u,
         uint256 m,
         uint256 a,
         uint256 r,
-        uint256 d
+        Pendle.ApproxParams calldata g,
```

Comment:
should we return these params from the BE? 🤔 

```
    struct ApproxParams {
        uint256 guessMin;
        uint256 guessMax;
        uint256 guessOffchain;
        uint256 maxIteration;
        uint256 eps;
    }
```

it seems like something that can be `static` for now, right?